### PR TITLE
[#358] Add alt-text for accessibility on installfest

### PR DESCRIPTION
**Pull request by tastycode**

### CODE REVIEW

File: sites/en/installfest/windows.step [hunk 1]
```diff
@@ -53,13 +46,13 @@ step "Update Git" do
 
   message "Run the installer, and it will ask you where you want to install it. Change it FROM C:\\Program Files\\Git  TO, C:\\RailsInstaller\\Git like the picture below."
 
-  img src: 'img/directory.png'
+  img src: 'img/directory.png', "Screenshot showing the step of installation where the location to install git is set"
```
Comment (tjgrathwell) [inline sites/en/installfest/windows.step]: Looks like you missed the `alt:` here, which is causing this page not to render / the tests to fail.

File: sites/en/installfest/windows.step [hunk 2]
```diff
@@ -53,13 +46,13 @@ step "Update Git" do
...
 
   message "It will warn you that the directory already exists. Click yes to install to that folder anyway."
 
   message "Click next twice, and select, Use Git from the Windows Command Prompt."
 
-  img src: 'img/command.png'
+  img src: 'img/command.png', "Screenshot showing location choice to use git from Windows Command Prompt"
```
Comment (tjgrathwell) [inline sites/en/installfest/windows.step]: also a missing `alt:` here

**Comment by tjgrathwell** [comment]
Glad to see some work on this @tastycode, fix up the test suite and it'l be ready to go.

**Comment by tastycode** [comment]
Sorry about the test-suite, I can't seem to run rake. I get a rather obscure deep stack trace. 

```
/Users/thomasdevol/.rvm/rubies/ruby-2.2.0/bin/ruby -I/Users/thomasdevol/.rvm/gems/ruby-2.2.0/gems/rspec-support-3.2.1/lib:/Users/thomasdevol/.rvm/gems/ruby-2.2.0/gems/rspec-core-3.2.0/lib /Users/thomasdevol/.rvm/gems/ruby-2.2.0/gems/rspec-core-3.2.0/exe/rspec --pattern spec/\*\*/\*_spec.rb --format d --color
/Users/thomasdevol/.rvm/gems/ruby-2.2.0/gems/wrong-0.7.1/lib/wrong/close_to.rb:12:in `<module:Wrong>': uninitialized constant Wrong::Date (NameError)
	from /Users/thomasdevol/.rvm/gems/ruby-2.2.0/gems/wrong-0.7.1/lib/wrong/close_to.rb:1:in `<top (required)>'
	from /Users/thomasdevol/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/thomasdevol/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/thomasdevol/.rvm/gems/ruby-2.2.0/gems/wrong-0.7.1/lib/wrong/helpers.rb:5:in `<top (required)>'
	from /Users/thomasdevol/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/thomasdevol/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/thomasdevol/.rvm/gems/ruby-2.2.0/gems/wrong-0.7.1/lib/wrong.rb:6:in `<top (required)>'
	from /Users/thomasdevol/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/thomasdevol/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/thomasdevol/.rvm/gems/ruby-2.2.0/gems/wrong-0.7.1/lib/wrong/adapters/rspec.rb:1:in `<top (required)>'
	from /Users/thomasdevol/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:128:in `require'
	from /Users/thomasdevol/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:128:in `rescue in require'
	from /Users/thomasdevol/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:39:in `require'
	from /Users/thomasdevol/workspace/docs/spec/spec_helper.rb:6:in `<top (required)>'
	from /Users/thomasdevol/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/thomasdevol/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/thomasdevol/workspace/docs/spec/app_deck_spec.rb:1:in `<top (required)>'
	from /Users/thomasdevol/.rvm/gems/ruby-2.2.0/gems/rspec-core-3.2.0/lib/rspec/core/configuration.rb:1226:in `load'
	from /Users/thomasdevol/.rvm/gems/ruby-2.2.0/gems/rspec-core-3.2.0/lib/rspec/core/configuration.rb:1226:in `block in load_spec_files'
	from /Users/thomasdevol/.rvm/gems/ruby-2.2.0/gems/rspec-core-3.2.0/lib/rspec/core/configuration.rb:1224:in `each'
	from /Users/thomasdevol/.rvm/gems/ruby-2.2.0/gems/rspec-core-3.2.0/lib/rspec/core/configuration.rb:1224:in `load_spec_files'
	from /Users/thomasdevol/.rvm/gems/ruby-2.2.0/gems/rspec-core-3.2.0/lib/rspec/core/runner.rb:97:in `setup'
	from /Users/thomasdevol/.rvm/gems/ruby-2.2.0/gems/rspec-core-3.2.0/lib/rspec/core/runner.rb:85:in `run'
	from /Users/thomasdevol/.rvm/gems/ruby-2.2.0/gems/rspec-core-3.2.0/lib/rspec/core/runner.rb:70:in `run'
	from /Users/thomasdevol/.rvm/gems/ruby-2.2.0/gems/rspec-core-3.2.0/lib/rspec/core/runner.rb:38:in `invoke'
	from /Users/thomasdevol/.rvm/gems/ruby-2.2.0/gems/rspec-core-3.2.0/exe/rspec:4:in `<main>'
/Users/thomasdevol/.rvm/rubies/ruby-2.2.0/bin/ruby -I/Users/thomasdevol/.rvm/gems/ruby-2.2.0/gems/rspec-support-3.2.1/lib:/Users/thomasdevol/.rvm/gems/ruby-2.2.0/gems/rspec-core-3.2.0/lib /Users/thomasdevol/.rvm/gems/ruby-2.2.0/gems/rspec-core-3.2.0/exe/rspec --pattern spec/\*\*/\*_spec.rb --format d --color failed
```

In the absence of being able to run rake, I did `git ls-files | grep '.\(rb\|step\)$'  | xargs -I % ruby -c %` instead.

**Comment by tjgrathwell** [comment]
Does `bundle exec rake` work?

Also, please squash the two commits into one so that all commits pass the tests.

**Comment by tastycode** [comment]
That did it! Thanks

**Comment by tjgrathwell** [comment]
:+1: sounds good!

**Pull request merged**